whois-nrtm4/src/main/java/net/ripe/db/nrtm4/publish/PublishableDeltaFile.java

```diff
@@ -1,5 +1,6 @@
 package net.ripe.db.nrtm4.publish;
```

Comment:
I suggest the package name `domain` for connsistency with the rest of Whois

Reply:
done